src/core/less/common/accessibility.less

```diff
@@ -0,0 +1,169 @@
+// Accessibility
+// =============
+
+//jquery.a11y start
+/*INVISIBLE ARIA-LABELS*/
+.aria-label {
+    position:absolute !important;
+    &.relative {
+        position:relative !important;
+    }
+    left:0px !important;
+    width:auto !important;
+    height:auto !important;
+    overflow:auto !important;
+    filter: progid:DXImageTransform.Microsoft.Alpha(Opacity=0); /*IE8*/
+    color: rgba(0,0,0,0) !important;
+    background: rgba(0,0,0,0) !important;
+    font-size: 1px !important;
+    padding:0 !important;
+    margin:0 !important;
+    line-height: normal !important;
+    z-index:1;
+}
+
+    /*FORCE ARIA-LABELS TO HIDE ON HIDDEN*/
+    .aria-label.aria-hidden {
+        display:none !important;
+    }
+
+/*HIDDEN FOCUS GUARD*/
+#a11y-focusguard {
+    left:0px !important;
+    bottom:0px !important;
+    width:auto !important;
+    height:auto !important;
+    overflow:auto !important;
+    filter: progid:DXImageTransform.Microsoft.Alpha(Opacity=0); /*IE8*/
+    color: rgba(0,0,0,0) !important;
+    background: rgba(0,0,0,0) !important;
+    font-size: 1px !important;
+    padding:0 !important;
+    margin:0 !important;
+    line-height: normal !important;
+    z-index:1;
+
+    /*IPAD: Stop accessibility cursor focusing until end of page;*/
+    .touch & { 
+        position: relative !important;
+    }
+
+    /*DESTKOP: stop focus making window jump to bottom of page;*/
+    .no-touch & {
+        position: fixed !important;
+    }
+}
+
+/*HIDDEN FOCUS/ALERT ELEMENT */
+#a11y-selected, #a11y-selected * {
+    position:fixed !important;
+    left:0px !important;
+    bottom:0px !important;
+    width:auto !important;
+    height:auto !important;
+    overflow:auto !important;
+    filter: progid:DXImageTransform.Microsoft.Alpha(Opacity=0); /*IE8*/
+    color: rgba(0,0,0,0) !important;
+    background: rgba(0,0,0,0) !important;
+    font-size: 1px !important;
+    padding:0 !important;
+    margin:0 !important;
+    line-height: normal !important;
+    z-index:1;
+}
+
+/*FOCUS DISTRACTION*/
+#a11y-focuser {
+    position:fixed !important;
+    left:0px !important;
+    top:0px !important;
+    width:auto !important;
+    height:auto !important;
+    overflow:auto !important;
+    filter: progid:DXImageTransform.Microsoft.Alpha(Opacity=0); /*IE8*/
+    color: rgba(0,0,0,0) !important;
+    background: rgba(0,0,0,0) !important;
+    font-size: 1px !important;
+    padding:0 !important;
+    margin:0 !important;
+    line-height: normal !important;
+    z-index:1;
+}
+
+.no-select {
+    -webkit-touch-callout: none;
```

Comment:
.no-select, .visibility-hidden and .display-none shouldn't be in here, they're not for accessibility

Reply:
Done.